### DISCUSSION TITLE
Search and codecopy

### DISCUSSION THREAD

**Discussion by CakeIsAfterMe** [nunocoracao/blowfish#1539] [Q&A]
Hi!

I'm not good at html and stuff, so I'd like to know what needs to be done to make "enableSearch" and "enableCodeCopy" values in params.toml work?
I mean, they work in localhost, but when uploading the site to Github Pages, they don't. 

If you know what the problem is, please help

**Comment by nunocoracao** [comment]
Have you configure the base url correctly? I would search for that in the docs and try it
